alerts/openshift-virtualization-operator/NodeStatusMaxImagesExceeded.md

```diff
@@ -0,0 +1,33 @@
+# NodeStatusMaxImagesExceeded
+
+## Meaning
+
+This alert fires when a node exceeds the configured maximum number of images
+reported in the node status (default: 50). Once this limit is reached, the
+scheduler might not be able to accurately schedule new pods because the reported
+image count is wrong.
+
+## Impact
+
+Pod scheduling might be imbalanced across nodes.
+
+## Diagnosis
+
+1. Obtain the image count for the node:
+
+   ```bash
+   $ oc get nodes <node> -o jsonpath='{range .status.images[*]}{.names}{"\n"}{end}' | wc -l
+   ```
+
+2. If image count is equal to the default (50) or your custom configured
+value, resolve the issue.
+
+## Mitigation
+
+As a workaround, you can edit the `nodeStatusMaxImages` parameter of the
```

Comment:
I'm not sure we need to call it a workaround. It's probably the easiest way to resolve the issue. So we can reword it like this:

You can either increase the `nodeStatusMaxImages` value of the `KubeletConfig` object or set its value to `-1` to disable the maximum image limit.